code/game/objects/items/ego_weapons/special.dm

```diff
@@ -68,3 +70,339 @@
+	icon = 'icons/obj/tools.dmi'
+	icon_state = "crowbar"
+	lefthand_file = 'icons/mob/inhands/equipment/tools_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/tools_righthand.dmi'
+	usesound = 'sound/items/crowbar.ogg'
+	flags_1 = CONDUCT_1
+	slot_flags = ITEM_SLOT_BELT
+	force = 10
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	var/faction_bonus_force = 15
+	var/nemesis_factions = list("green_ordeal") // UNLOCK YOUR COVER AND STATE LAWS!
+	throwforce = 12
+	w_class = WEIGHT_CLASS_SMALL
+	custom_materials = list(/datum/material/iron=50)
+	drop_sound = 'sound/items/handling/crowbar_drop.ogg'
+	pickup_sound =  'sound/items/handling/crowbar_pickup.ogg'
+
+	attack_verb_continuous = list("attacks", "bashes", "batters", "bludgeons", "whacks")
+	attack_verb_simple = list("attack", "bash", "batter", "bludgeon", "whack")
+	tool_behaviour = TOOL_CROWBAR
+	toolspeed = 1
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, RAD = 0, FIRE = 50, ACID = 30)
+	var/force_opens = FALSE
+
+/obj/item/ego_weapon/crowbar/attack(mob/living/target, mob/living/carbon/human/user)
+	var/nemesis_faction = FALSE
+	if(LAZYLEN(nemesis_factions))
+		for(var/F in target.faction)
+			if(F in nemesis_factions)
+				nemesis_faction = TRUE
+				force += faction_bonus_force
+				break
+	. = ..()
+	if(nemesis_faction)
+		playsound(target.loc,'sound/effects/wounds/pierce1.ogg', rand(30,50), TRUE) // Play funny sound to signify additional damage.
+		force -= faction_bonus_force
+
+/obj/item/ego_weapon/crowbar/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is claiming worker's compensation with [src]! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	playsound(loc, 'sound/weapons/genhit.ogg', 50, TRUE, -1)
+	return (BRUTELOSS)
+
+//////////////////////////////Hardhead Shotgun//////////////////////////////
+
+/obj/item/gun/ego_gun/hardened_shotgun
+	name = "appropiated shotgun"
+	desc = "Company guards hesitated to fire bullets coming out of their own pay, and quickly abandoned both guns and duty."
+	worn_icon_state = null
+	lefthand_file = 'icons/mob/inhands/weapons/64x_guns_left.dmi'
+	righthand_file = 'icons/mob/inhands/weapons/64x_guns_right.dmi'
+	icon_state = "shotgun"
+	inhand_icon_state = "shotgun"
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	ammo_type = /obj/item/ammo_casing/caseless/hardhead_shotgun
+	weapon_weight = WEAPON_HEAVY
+	fire_delay = 10
+	recoil = 1		//Shakes your screen
+	fire_sound = 'sound/weapons/gun/shotgun/shot_auto.ogg'
+	w_class = WEIGHT_CLASS_BULKY
+	force = 10
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	flags_1 =  CONDUCT_1
+	slot_flags = ITEM_SLOT_BACK
+
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 30,
+							JUSTICE_ATTRIBUTE = 40
+							)
+
+//////////////////////////////Hardhead Clipboard//////////////////////////////
+
+/obj/item/ego_weapon/clipboard
```

Comment:
You don't have to copy the entire goddamn clipboard code to this. Just keep it a weapon, remove all the other stuff.